Modules/Numerics/Optimizersv4/wrapping/itkGradientDescentLineSearchOptimizerv4.wrap

```diff
@@ -0,0 +1,7 @@
+set(WRAPPER_AUTO_INCLUDE_HEADERS OFF)
+itk_wrap_include("itkGradientDescentLineSearchOptimizerv4.h")

```

Comment:
Does it still work if you remove the first 2 lines?

Reply:
No. I tried it and I also found an explanation in the ITK software guide. The class has name Template in it but the header does not have that and the default header including fails because of that.

**By default, the class wrapping macros include a header whose filename corresponds to the name of
the class to be wrapped according to ITK naming conventions. To override the default behavior, set
the CMake variable WRAPPER AUTO INCLUDE HEADERS to OFF before calling itk wrap class.**

Reply:
Thanks